### PR TITLE
Added the expectation of the OnHMILevel(NONE)

### PR DESCRIPTION
**Pull request by GetmanetsIrina**
ATF Test Scripts to check #[3073](https://github.com/smartdevicelink/sdl_core/issues/3073)

This PR is **ready** for review.

### Summary
Added the expectation of the OnHMILevel(NONE) notification according to https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0211-ServiceStatusUpdateToHMI.md

### ATF version
develop

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/API/ServiceStatusUpdateToHMI/001_1_Video_service_protected.lua [hunk 1]
```diff
@@ -41,6 +41,9 @@ function common.onServiceUpdateFunc(pServiceTypeValue)
     { serviceEvent = "REQUEST_RECEIVED", serviceType = pServiceTypeValue, appID = common.getHMIAppId() },
     { serviceEvent = "REQUEST_ACCEPTED", serviceType = pServiceTypeValue, appID = common.getHMIAppId() })
   :Times(2)
+
```
Comment (dboltovskyi) [inline test_scripts/API/ServiceStatusUpdateToHMI/001_1_Video_service_protected.lua]: @GetmanetsIrina Please update description with appropriate expectation on this notification

File: test_scripts/API/ServiceStatusUpdateToHMI/020_Video_service_protected_PTU_FAILED_TimeOut_2_failed_external.lua [hunk 2]
```diff
@@ -53,6 +53,10 @@ function common.onServiceUpdateFunc(pServiceTypeValue)
     end)
   :Times(2)
   :Timeout(timeout)
+
+  common.getMobileSession():ExpectNotification("OnHMIStatus",
+    { hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN" })
+  :Timeout(timeout)
```
Comment (dboltovskyi) [inline test_scripts/API/ServiceStatusUpdateToHMI/020_Video_service_protected_PTU_FAILED_TimeOut_2_failed_external.lua]: @GetmanetsIrina Please notice script invokes `startServiceWithOnServiceUpdate_PTU_FAILED()` function 2 times.
App will be moved to NONE during 1st execution and stayed in NONE during 2nd one.
Propose to activate app before 2nd execution.
This also applicable for `021` and `022` scripts

Comment (GetmanetsIrina) [reply test_scripts/API/ServiceStatusUpdateToHMI/020_Video_service_protected_PTU_FAILED_TimeOut_2_failed_external.lua]: @dboltovskyi, please find the update in 962e6c2

File: test_scripts/API/ServiceStatusUpdateToHMI/001_1_Video_service_protected.lua (hunk 1 above)
Comment (GetmanetsIrina) [reply test_scripts/API/ServiceStatusUpdateToHMI/001_1_Video_service_protected.lua]: @dboltovskyi, please find the update in 962e6c2

**Comment by theresalech** [comment]
@GetmanetsIrina if this PR is ready for Livio review, can you please update the description to `This PR is **not ready** for review.`?

**Comment by dboltovskyi** [comment]
@theresalech Description is updated

**Pull request merged**